static/src/javascripts/projects/common/modules/identity/wizard.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+/* eslint-disable no-underscore-dangle, react/sort-comp */
+
+import bean from 'bean';
+import $ from 'lib/$';
+import { Component } from 'common/modules/component';
+
+const stepClassname = 'manage-account-wizard__step';
+const nextButtonElClassname = 'js-manage-account-wizard__next';
+const prevButtonClassname = 'js-manage-account-wizard__prev';
+export const containerClassname = 'manage-account-wizard';
+
+export class Wizard extends Component {
```

Comment:
I think for now this can stay, but for the future: we try to encourage the `singleton` pattern over classes in order to reduce complexity. `Component` is one of those things everyone hates and should not exist, because it's really a special case in the codebase ...